includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -1209,4 +1211,75 @@ public function track_job_submission( $post_id ) {
 			]
 		);
 	}
+
+	/**
+	 * Checks if 'extend' is set as action.
+	 *
+	 * @since $$next-version$$
+	 * @return bool
+	 */
+	public function is_extend_action() {
+		return isset( $_GET['action'] ) && sanitize_text_field( wp_unslash( $_GET['action'] ) ) === 'extend';
+	}
+
+	/**
+	 * Handle steps for extending a listing's expiry.
+	 *
+	 * @access private
+	 * @param array $steps
+	 * @return array
+	 */
+	public function maybe_extend_job_expiry( $steps ) {
```

Comment:
Maybe we could name this `remove_edit_steps` to make it more clear?

Reply:
Addressed with the other renamings to 'renew' in https://github.com/Automattic/WP-Job-Manager/pull/2422/commits/fb1244e94979c13c938fca46a70d985e594e3742